model.py

```diff
@@ -8687,6 +8698,7 @@ class DeliveryMechanism(Base, HasFullTableCache):
     STREAMING_TEXT_CONTENT_TYPE = u"Streaming Text"
     STREAMING_AUDIO_CONTENT_TYPE = u"Streaming Audio"
     STREAMING_VIDEO_CONTENT_TYPE = u"Streaming Video"
+    PERIODICAL_STREAMING_TEXT_CONTENT_TYPE = u"Periodical Streaming Text"
```

Comment:
I'm pretty sure you should use STREAMING_TEXT_CONTENT_TYPE here. Whether the text being streamed is a periodical or something else isn't relevant here, just as `text/html` doesn't make a distinction between different types of web pages.

Reply:
ok, perfect